src/MongoBulkDataMigration.ts

```diff
@@ -280,6 +285,10 @@ export default class MongoBulkDataMigration<TSchema extends Document>
   }
 
   async rollback(): Promise<BulkOperationResult> {
+    if (!this.options.rollbackable) {
+      return { ok: 1 } as any;
```

Comment:
@JulienBara-360 Could be meaningful to have additionally
```
logger.warn("Calling rollback() on a non rollbackable script");
```

If someone calls this by mistakes, it will be less surprising

Reply:
@pp0rtal 
Great idea! Done here: https://github.com/360Learning/mongo-bulk-data-migration/pull/8/commits/d33372c7fe06677c0fb74f7d64a57d7b034c3a71
Thinking about it, we could even have thrown. Not sure how it can impact negatively a flow. We are not supposed to call it.
But let's keep it this way for now. It should for my use case.